src/js/disability-benefits/526EZ/config/form.js

```diff
@@ -183,6 +213,8 @@ const formConfig = {
           title: (formData, { pagePerItemIndex }) => _.get(`disabilities.${pagePerItemIndex}.diagnosticText`, formData),
           path: 'supporting-evidence/:index/evidence-type',
           showPagePerItem: true,
+          // TODO: Remove the nested `disability` when we flatten that
+          itemFilter: (item) => _.get('disability.view:selected', item),
```

Comment:
`disability` will get removed in the process of resolving merge conflicts, so will disappear before ratedDisabilities is merged to master?

Reply:
Ah, right. I have to fix all those `TODO`s now. Forgot about that, thanks. :+1: